templates/basic-ts/test/index.test.ts

```diff
@@ -5,6 +5,8 @@ import nock from "nock";
 // Requiring our app implementation
 import myProbotApp from "../src/index.js";
 import { Probot, ProbotOctokit } from "probot";
+// Requiring the OctokitOptions for ProbotOctokit.defaults()
+import {OctokitOptions} from "probot/lib/types";
```

Comment:
You shouldn't use internal entrypoints like this.

TypeScript can infer these types anyways.

In a future major version, these won't be accesssible